### ISSUE TITLE
Unsupported WooCommerce version issue

### ISSUE THREAD

**Issue by dominikj111** [g2a-official/g2a-pay-woocommerce#2]
In the WooCommerce 3.5.2 and 3.5.3 status page, I'm getting error message on the g2apay plugin download from G2A Pay portal (tried both available versions 3.1.2 and 2.5).

The error message is saying: 
**WooCommerce G2A Pay Payment Gateway - by G2A Team – 1.5 – Not tested with the active version of WooCommerce**

**Comment by Chaython** [comment]
where?
Just download the plugin from this git

**Comment by domino2** [comment]
Yeah, this is what I can live with, but what is worse that paid orders are not marked as completed, even they are completed and money are taken, what we can see in the G2A payments list. So I was thinking that my settings is wrong, but I have set correctly API Hash, secret key and the IPN so what I should to check next? 

Thanks for a response.

**Comment by Chaython** [comment]
It usually completes for me once payment is verified; there's only been 10 / 300 orders where it never completed
it's usually because the token timed out [a slow payment method like bank transfer]

**Comment by domino2** [comment]
Hello, after a while, I'm responding back. I've done fix for our project and it is working as expected. You can check it your self or we can merge it to _g2a-official_ if wanted.
